src/main/java/cat/udl/eps/entsoftarch/webingogeiapi/domain/Invitation.java

```diff
@@ -22,6 +17,7 @@
     private long id;
 
     @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    @Column(name = "created_at")
```

Comment:
Why is the name of the field in the database changed? Is there any requirement to do so?

Reply:
It's for good practices, java objects names are written in CamelCase (createdAt), opposite in bbdd the names are written in Snake Case (created_at). There are other names that we don't changed now because the other persons are working with it. When all the people work with angular, we will try change it.

If we do a project collaborative better if we  respect good practices of programming.